### DISCUSSION TITLE
How can I get Pair Extraordinaire

### DISCUSSION THREAD

**Discussion by newstable** [community/community#36443] [Other Feature Feedback, Questions, & Ideas] [answered]
I wanna get Pair Extraordinaire achievement but I don't know how can I get this achievement.
so I need help of expert.
Thank you

**Comment by driica** [comment, marked as answer]
Hello, I nominate a channel where he talks about it, but it's in BR, but it has English subtitles. Link: https://youtu.be/G36D8fE4Pzo

**Reply by newstable** [reply]
Thank you for your answer.
